src/js/post-911-gib-status/containers/StatusPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { Link } from 'react-router';
+
+import FormTitle from '../../common/schemaform/FormTitle';
+
+import { getEnrollmentData } from '../actions/post-911-gib-status';
+import EnrollmentHistory from '../components/EnrollmentHistory';
+import UserInfoSection from '../components/UserInfoSection';
+
+class StatusPage extends React.Component {
+  componentDidMount() {
+    this.props.getEnrollmentData();
+  }
+
+  render() {
+    // TODO: change the service name below from "user-profile" to
+    // something like "post-911-gib-status" once its defined in vets-api
+    const { enrollmentData } = this.props;
+
+    return (
+      <div>
+        <FormTitle title="Post-9/11 GI Bill Status"/>
+        <div className="va-introtext">
+          <p>
+            View your Post-9/11 GI Bill enrollment information below. This is the same information
+            in your Certificate of Eligibility (COE) letter. In lieu of a COE letter, you can
+            print a copy of this screen for benefit and eligibility verification.
+          </p>
+        </div>
+        <div className="section-line">
+          <Link to="/print" target="_blank" className="usa-button-primary">
+            Print This Page as COE
+          </Link>
+        </div>
+        <UserInfoSection enrollmentData={enrollmentData} showCurrentOfAlert/>
```

Comment:
It doesn't come from anywhere; `showCurrentOfAlert` is just a prop passed to `UserInfoSection`. If the prop is passed, the value is `true`; if it's not passed, the value is `false`. So by including this prop, the value of `showCurrentOfAlert` once within `UserInfoSection` is `true`. I originally had `showCurrentOfAlert={true}`, but then React warned me that was unnecessary :)